include/envoy/network/connection_handler.h

```diff
@@ -38,10 +38,12 @@ class ConnectionHandler {
   virtual void decNumConnections() PURE;
 
   /**
-   * Adds a listener to the handler.
+   * Adds a listener to the handler. and replace the existing listener.
+   * @param overrided_listener tag of the existing listener. nullopt if no previous listener.
```

Comment:
`overridden_listener` here and elsewhere.

Reply:
:+1: